pkg/gofr/gofr.go

```diff
@@ -88,6 +88,17 @@ func New() *App {
 	app.httpServer.certFile = app.Config.GetOrDefault("CERT_FILE", "")
 	app.httpServer.keyFile = app.Config.GetOrDefault("KEY_FILE", "")
 
+	// Add Default routes
+	app.add(http.MethodGet, "/.well-known/health", healthHandler)
+	app.add(http.MethodGet, "/.well-known/alive", liveHandler)
```

Comment:
Just a question, not about your changes in this PR

Are these well known URI somehow standard?

The only documentation I found was the gofr documentation

https://gofr.dev/docs/advanced-guide/monitoring-service-health

The documentation doesn't talk about it

I don't find them also on Wikipedia

https://en.wikipedia.org/wiki/Well-known_URI

Doesn't it mean gofr created its own "standard", but that is the only one to use?

I'm unsure I would have used .well-known URI for these

Unless there is an rfc or other project using this

Reply:
Hi @ccoVeille 

Thank you for your insightful question regarding the `.well-known` URIs.

You are correct that these URIs are not part of an official RFC or standard for the specific use cases we've implemented in GoFr. We developed these endpoints to streamline access to important API information such as health status, swagger documentation, and more. Our goal was to provide a consistent and convenient way for developers to obtain vital information about their APIs without the need for additional authentication.

While the .well-known URIs in GoFr may not align with a globally recognized standard, we believe they offer a practical solution for common development needs. We appreciate your feedback and will continue to consider how we can improve our approach to be more in line with industry standards and best practices.

Please let us know if you have any further questions or suggestions. We're here to help!

Reply:
I think this information is solid. I would suggest to add them in the code documentation but also in the official documentation

Reply:
Sure @ccoVeille will add this to the documentations also

Reply:
Side remark, I would have used a gofr prefix in the URI then
Something like /.well-known/gofr

But I assume it's either too late and opinionated